src/index.html

```diff
@@ -6,12 +6,83 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
+    <link rel="stylesheet" href="style.css">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text" id="surname" 
+        name="surname" autocomplete="off"
+        required><br>
```

Comment:
```suggestion
        required>
```
don't use `<br />`, if you don't need to make a text linebreak 😉